tests/lib/test_atomicals_utils.py

```diff
@@ -459,3 +460,116 @@ def test_calculate_expected_bitwork_base():
     assert(calculate_expected_bitwork('abcdefe', 33000, 1000, 1, 127) == 'abcdefe000')
     assert(calculate_expected_bitwork('abcdefe', 33000, 1000, 3, 127) == 'abcdefe0000000.2')
      
+def test_calculate_expected_bitwork_rollover():
+    
+    assert(calculate_expected_bitwork('888888888888', 49995, 3333, 1, 64) == '8888.15')
```

Comment:
Consider adding cases to guard increments other than `1`.

Reply:
please also consider adding cases for more general cases like: 8888.2, 8888.3, 8888.8, 8888.9

Reply:
Done, added a variety and using an array-loop